src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -103,7 +98,7 @@ public CompletableFuture<BindingResult<CompositePurchaseOrder>> map(Map<Mapping.
   }
 
   private void mapPurchaseOrderWorkflow(BindingResult<CompositePurchaseOrder> bindingResult) {
-    EnumSet<Mapping.Field> pendingSet = EnumSet.of(BILL_TO, SHIP_TO, LINKED_PACKAGE, SUFFIX, PREFIX);
+    EnumSet<Mapping.Field> pendingSet = EnumSet.of(LINKED_PACKAGE, SUFFIX, PREFIX);
```

Comment:
Sorry that I have not catched this in previous PR: https://github.com/folio-org/acq-models/pull/395/files
Basically we should not delete fields SHIP_TO, BILL_TO, we need delete only translators 
![image](https://user-images.githubusercontent.com/25097693/186668710-83113ca1-7502-40b0-b46d-9d44fe9fc501.png)
So we need to delete LOOKUP_BILL_TO, LOOKUP_SHIP_TO translators and remain only LOOKUP_CONFIG_ADDRESS translator

Reply:
Here is REST request to check what translators presented: {{protocol}}://{{url}}:{{okapiport}}/gobi/orders/mappings/translators

Reply:
sure, fixed.